requirements_test_min.txt

```diff
@@ -1,3 +1,4 @@
+-e .
```

Comment:
I'm not sure this is a good idea. We do pin some of the requirements of `pylint` later on (mainly `astroid` and `isort`). With this line `pip` will probably install pylint first and then have to go back and install different versions of some dependencies.

Reply:
I did a quick test to see how the pip dependency resolver would work. I seems my worries were unfounded. It works, just like the setup does now.

With that, `pip install -r requirements_test.txt` should be enough to setup the whole environment.